lighthouse-core/lib/url-shim.js

```diff
@@ -11,20 +11,14 @@
 
 /* global self */
 
+/** @typedef {import('url').URL} OriginalURL */
```

Comment:
@brendankenny wasn't sure how to fix this but `new Window["URL"]` isn't compatible with itself.